src/main/java/com/kuljava/swiatwsi/rawmaterials/RawMaterialsRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.kuljava.swiatwsi.rawmaterials;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import javax.transaction.Transactional;
+
+@Repository
+public interface RawMaterialsRepository extends JpaRepository<RawMaterials, Long> {
+
+    @Transactional
+    @Modifying
+    @Query("UPDATE RawMaterials r SET r.woodQuantity = r.woodQuantity + ?1, r.clayQuantity = r.clayQuantity + ?2, r.ironQuantity = r.ironQuantity + ?3")
```

Comment:
For such easy operation, there probably exists method in jparepository/crudrepository. Maybe save()?

Reply:
Do you mean default method using save() instead of query?

Reply:
In JPA repository already exists method save. You could probably use it instead of creating Your own method, especially for operations simple as this one

Reply:
But using query I can simply update all RawMaterials. Can I do it using save?